injects/sampler.js

```diff
@@ -23,6 +23,26 @@ const processStat = new ProcessStat(parseInt(
   process.env.NODE_CLINIC_DOCTOR_SAMPLE_INTERVAL, 10
 ))
 
+function checkForTranspiledCode (filename) {
+  const readFile = fs.readFileSync(filename, 'utf8')
+  const regex = /function\s+(?<functionName>\w+)/g
+  let matchedObj; let longerThanLimit = false
+
+  while ((matchedObj = regex.exec(readFile)) !== null) {
+    // Avoid infinite loops with zero-width matches
+    if (matchedObj.index === regex.lastIndex) {
+      regex.lastIndex++
+    }
+    // Loop through results and check length of fn name
+    matchedObj.forEach((match, groupIndex) => {
+      if (groupIndex !== 0 && match.length > 3) {
```

Comment:
I think for the _server-side_ transpiled code that we're mostly interested in, minification is quite rare. The code would typically be transpiled with babel or typescript only. Another possible heuristic is looking for source map comments at the end of the file:
https://docs.google.com/document/d/1U1RGAehQwRypUTovF1KRlpiOFze0b-_2gc6fAH0KY0k/edit#heading=h.lmz475t4mvbx
```
//# sourceMappingURL=<url>
```

Reply:
I see. Some Clinic issues were reported by using frameworks like Nuxt which does have minified output for the server side. Checking for a source map is also planned.

Reply:
Gotcha. sounds good